.github/workflows/conformance-kind.yaml

```diff
@@ -117,25 +117,6 @@ jobs:
           pkill -f "cilium.*hubble.*port-forward|kubectl.*port-forward.*hubble-relay"
           cilium uninstall --chart-directory=install/kubernetes/cilium --wait
 
-      - name: Install Cilium with encryption
```

Comment:
If the intention is to disable tests temporarily, then it makes more sense to add the line below
```
if: ${{ false }}
``` 
And please add a comment if there is an issue that tracks this, or anything relevant.